app/forms/author_form.rb

```diff
@@ -3,15 +3,15 @@
 # Form for an author
 class AuthorForm < ApplicationForm
   attribute :first_name, :string
-  validates :first_name, presence: true, if: -> { person? && deposit? }
+  validates :first_name, presence: true, on: :deposit, if: -> { person? }
```

Comment:
This batch of changes separate data concerns ("Is the form a person-like author or an org-like author?") from context concerns ("Are we validating a draft or a deposit?").